go/libraries/doltcore/sqle/enginetest/dolt_engine_test.go

```diff
@@ -203,7 +203,7 @@ func TestAddColumn(t *testing.T) {
 }
 
 func TestModifyColumn(t *testing.T) {
-	t.Skip("Type changes aren't supported")
+	t.Skip("Type changes not supported")
```

Comment:
These should work though? Removing this skip doesn't allow this to pass?

Reply:
No, tried it. I think it changes a primary key column, but I didn't dig in much.